setup.py

```diff
@@ -194,6 +194,7 @@
 import setuptools.command.build_ext
 import setuptools.command.install
 import distutils.command.clean
+import setuptools.command.sdist
```

Comment:
Note that `python setup.py sdist` doesn't work (I opened a separate issue for that), so you may want to take this bit out for now and put it as a comment on the `sdist` repo. Getting the full license listing and stuffing them in sdists/wheels is best separated. 

The wheel change should probably be done in https://github.com/pytorch/builder

Reply:
reverted the setup.py changes. Once this is merged, it can be used in the other repos.